rllib/offline/estimators/off_policy_estimator.py

```diff
@@ -91,6 +192,21 @@ def check_action_prob_in_batch(self, batch: SampleBatchType) -> None:
                 "`off_policy_estimation_methods: {}` to disable estimation."
             )
 
+    def _compute_action_probs(self, batch: SampleBatch):
+        log_likelihoods = compute_log_likelihoods_from_input_dict(self.policy, batch)
```

Comment:
put these 2 lines in an else clause so we don't spend time on them if epsilon greedy is specified?

Reply:
These two should run regardless of eps? you modify new_prob in new_eps case?